sql/connect/common/src/main/protobuf/spark/connect/expressions.proto

```diff
@@ -483,11 +483,15 @@ message SubqueryExpression {
   // (Optional) Options specific to table arguments.
   optional TableArgOptions table_arg_options = 3;
 
+  // (Optional) IN subquery options.
+  optional InSubqueryOptions in_subquery_options = 4;
```

Comment:
You can also just reference the IN expression directly here. Or is there future work that requires a separate message?

Reply:
> is there future work that requires a separate message?

No, I just followed the way of `TableArgOptions`.

Updated.